### PR TITLE
DateRangeBox: add the Overview demo description

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: JSDemos/Demos/DateRangeBox/Overview/description.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The [DateRangeBox](/Documentation/ApiReference/UI_Components/dxDateRangeBox/) component allows users to enter or pick dates in a specified range. This demo illustrates the following DateRangeBox properties:
```
Comment (Abadzhev) [inline JSDemos/Demos/DateRangeBox/Overview/description.md]:
```suggestion
The [DateRangeBox](/Documentation/ApiReference/UI_Components/dxDateRangeBox/) component allows users to select a date range. The component includes input boxes for start/end dates and a date-picker dropdown. This demo shows how you can use the following DateRangeBox settings:
```

File: JSDemos/Demos/DateRangeBox/Overview/description.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- [value](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#value)     
+An array where you can specify the selected range (start and end dates). The DateRangeBox allows you to define the start and end dates separately. For this purpose, use the [startDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#startDate) and [endDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#endDate) properties.
```
Comment (Abadzhev) [inline JSDemos/Demos/DateRangeBox/Overview/description.md]:
```suggestion
An array where you can specify the selected range (start and end dates). The DateRangeBox also allows you to define the start and end dates separately. For this purpose, use the [startDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#startDate) and [endDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#endDate) properties instead.
```

File: JSDemos/Demos/DateRangeBox/Overview/description.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- [displayFormat](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#displayFormat)        
+A date display format. You can use one of the [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type) or specify a [custom format](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String). This demo illustrates the latter.
+
+- [disabled](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#disabled)        
+Specifies whether the DateRangeBox responds to user interaction.
+
+- [showClearButton](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#showClearButton)        
+This property allows you to display the button that clears the DateRangeBox **values**.
```
Comment (Abadzhev) [inline JSDemos/Demos/DateRangeBox/Overview/description.md]: Other descriptions don't start with "this property", so let's continue the same pattern for consistency. 

 ```suggestion
Displays the button that clears DateRangeBox **values**.
```

File: JSDemos/Demos/DateRangeBox/Overview/description.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+- [multiView](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#multiView)     
+Allows allows you to pick the start or end dates in a single or multiple calendars.
```
Comment (Abadzhev) [inline JSDemos/Demos/DateRangeBox/Overview/description.md]:
```suggestion
Switches between a single-month and two-month dropdown calendar.
```

File: JSDemos/Demos/DateRangeBox/Overview/description.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+The [DateRangeBox](/Documentation/ApiReference/UI_Components/dxDateRangeBox/) component allows users to select a date range. The component includes input boxes for start/end dates and a date-picker dropdown. This demo shows how you can use the following DateRangeBox settings:
```
Comment (dirk-pieterse) [inline JSDemos/Demos/DateRangeBox/Overview/description.md]:
```suggestion
The [DateRangeBox](/Documentation/ApiReference/UI_Components/dxDateRangeBox/) component allows users to select a date range. The component includes input boxes for start/end dates and a drop-down date picker. This demo shows how you can use the following DateRangeBox settings:
```

**Review by dirk-pieterse — DISMISSED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**